src/AggregateRootWithAggregates.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EventSauce\EventSourcing;
+
+use SplObjectStorage;
+
+trait AggregateRootWithAggregates
+{
+    use AggregateRootBehaviour, AggregateAppliesKnownEvents {
+        AggregateAppliesKnownEvents::apply as applyOnAggregateRoot;
```

Comment:
The aliasing is required unfortunately. The reconstruction code only knows about `apply`, this functionality replaces that but still needs to sub-dispatch to the identical implementation. It could have been a copy+paste otherwise. I'm not too fuzzed about this. I agree it's a little confusing, but it's not so bad that I want to change it.